services/smtp.go

```diff
@@ -30,9 +30,97 @@
  */
 package services
 
-/*
-// SMTP is a placeholder
-func (d *lowDirector) SMTP() func(net.Conn) {
-	return d.Echo()
+import (
+	"crypto/tls"
+	"net"
+
+	"github.com/honeytrap/honeytrap/event"
+	"github.com/honeytrap/honeytrap/pushers"
+)
+
+var (
+	_           = Register("smtp", SMTP)
+	receiveChan = make(chan Message)
+	serverCert  = "/etc/ssl/certs/mailserver.pem"
```

Comment:
Could you look at services/ssh to use the new storage for saving certificates? And can we generate new certificates on startup, just like SSL?